app/javascript/src/components/Notes/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import Card from "components/Notes/Card";
+import { NOTES_LIST } from "components/Notes/constants";
+import SidePanel from "components/Notes/SidePanel";
+
+const NoteListing = () => (
+  <div className="flex flex-row">
+    <SidePanel title="Notes" />
+    <div className="flex flex-col">
+      <Header
+        actionBlock={<Button className="mx-2" label="Add Note +" />}
+        menuBarToggle={function noRefCheck() {}}
```

Comment:
Carefully go through https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663060378971029 and fix the issue here. Take your time to understand the review comments and learn them.

Reply:
After going through the screenshots , I understand I should use arrow function instead of function keyword , will fix this .